package.json

```diff
@@ -105,7 +104,8 @@
     "react": ">=17.0.1",
     "react-dom": ">=17.0.1",
     "react-redux": ">=7.2.2",
-    "redux": "^4.1.2"
+    "redux": "^4.1.2",
+    "styled-components": ">=5"
```

Comment:
why >= instead of ^?

Reply:
following the example of the styled-components docs: 
> Also, note that in the peerDependencies section the version string has been made a more permissive >= n. This allows future versions of styled-components to work automatically and you can simply narrow the range with a patch update to your library if a breaking change is eventually added.

Reply:
`^` restricts to that major version, while `>=` allows higher major versions.

If a library does not use breaking changes, this is desirable, to allow the consumer to upgrade the dependency and still use (in this case) the widget. This is done with React a lot (because the breaking changes are usually more transparent to consumers).